loops-and-arrays.js

```diff
@@ -6,15 +6,19 @@
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
+console.log (names);
+for (let i = 0; i < names.length; i ++) {
+    console.log(names[i] + "je")
```

Comment:
De array wordt hier niet aangepast, de console.log voegt tijdelijk "je" toe aan de terminal. Dit kan je oplossen door de Array "names" te veranderen in een let (Zodat je hem mag overschrijven), en dan kan je daarna :

names[i] = names[i] + "je" ;

in de code gebruiken. Als je dan de tweede keer de names aanroept dan krijg je een array die geüpdatet is!

Reply:
Probeer nog even de andere opdrachten ook te veranderen zodat het het weergeeft in een Array, Maar het ziet er goed uit!